src/main/java/pl/edu/mimuw/Bank.java

```diff
@@ -1,25 +1,134 @@
 package pl.edu.mimuw;
 
-import java.util.List;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.Calendar;
 
 public class Bank {
 
-  @Override
-  public String toString() {
-    throw new IllegalStateException("TODO: get information about bank and all its clients (use StringBuilder)");
-  }
+    private final ArrayList<BankClient> allClients = new ArrayList<>();
```

Comment:
tak zawezasz sobie to co moze stac sie w roznych konstruktorach - raczej inizjalizujmy tam pola